pkg/policyprovider/configpolicy/configpolicy_test.go

```diff
@@ -80,3 +86,168 @@ func TestPolicyEnforcer_ContinueVerifyOnFailure(t *testing.T) {
 		t.Fatalf("For artifact types without a policy the default policy should be followed")
 	}
 }
+
+func TestPolicyEnforcer_OverallVerifyResult(t *testing.T) {
+
+	testcases := []struct {
+		configPolicyConfig map[string]interface{}
+		verifierReports    []interface{}
+		output             bool
+	}{
+		{
+			// no artifact policies or verifier reports
+			configPolicyConfig: map[string]interface{}{
+				"name":                         "configPolicy",
+				"artifactVerificationPolicies": map[string]types.ArtifactTypeVerifyPolicy{},
+			},
+			verifierReports: []interface{}{},
+			output:          true,
+		},
+		{
+			// no artifact policies
+			configPolicyConfig: map[string]interface{}{
+				"name":                         "configPolicy",
+				"artifactVerificationPolicies": map[string]types.ArtifactTypeVerifyPolicy{},
+			},
+			verifierReports: []interface{}{
+				vr.VerifierResult{
+					Subject:      "",
+					IsSuccess:    true,
+					ArtifactType: "application/vnd.cncf.notary.v2",
+				},
+			},
+			output: true,
```

Comment:
there has been some discussion on fail open or fail close. Looks like in these examples, we are failing open. Should this be property that can be configured?

Reply:
yeah that's a good point. I guess it will only be applicable in the scenario where no policies are specified. Right now it'll fail open. The only concern is that will this cause configuration confusion if we add another field? Would the user understand what the extra setting is actually specifying? (Maybe we set a default value for it so it isn't specified for most users)

Reply:
Jimmy mentioned the gatekeeper can be configured to specify fail open vs close. We are property is similar enough , customer might be able to pickup the pattern easily. Could you look into how to specify that in opa/gatekeeper?

Reply:
sounds good. I'll look into it

Reply:
thanks @akashsinghal, this thread might have some relevant info. https://open-policy-agent.github.io/gatekeeper/website/docs/failing-closed/

Reply:
Discussed offline that we can consider adding a `failurePolicy` to the config in the future https://github.com/deislabs/ratify/issues/167